examples/crowdfunding/test/crowdfundingTest.js

```diff
@@ -107,24 +111,24 @@ describe('Crowdfunding Tests', function () {
     assert.deepEqual(getGlobal('Total'), 0n);
 
     // update application with correct escrow account address
-    let appArgs = [addressToPk(escrow.address)]; // converts algorand address to Uint8Array
+    let appArgs = [addressToPk(lsig.address())]; // converts algorand address to Uint8Array
```

Comment:
Instead of using `lsig.address()`, we can declare a constant:
```
const escrowAddr = lsig.address();
```
It's more intuitive

Reply:
ok